cloudformation/master-server-substack.cfn.yaml

```diff
@@ -219,6 +221,9 @@ Conditions:
     - !Equals
       - !Ref 'UpdateWaiterFunctionArn'
       - NONE
+  MasterPublicIp: !Equals
```

Comment:
This we can rename to `UseMasterPublicIp`

Reply:
Done